.travis.yml

```diff
@@ -13,7 +13,9 @@ matrix:
     - language: node_js
       node_js: "10"
       install: npm install -g eslint
-      script: eslint .
+      script:
+        - npm test
+        - eslint .
```

Comment:
Please don't do eslint .
This will lint all files in the directory. There is config too!

Eslinting is already included and executed as part of `npm test` so I'm pretty sure you can drop this line.

Reply:
We will do it your way but __eslint .__ generates output while none of that output appears when __npm test__ is run.  Are we sweeping any issues under the carpet?

Reply:
```
eslint . 
```
will run against the whole directory and use the default eslint config, so will hit errors and generate output. (When I do this I get 12805 problems (12805 errors, 0 warnings))

On the other hand
```
npm test
```
will make use of our config file and only generate output if there's a problem and all the previous tests have passed (due to usage of &&  - we check bundles and run stylelint beforehand).

I threw up a patch that breaks eslint in #1747 and I'm seeing the output in Travis as expected:
https://travis-ci.org/internetarchive/openlibrary/jobs/470173700

What am I misunderstanding?

Reply:
Sounds like you got it right and I got it wrong.